### PR TITLE
FOSSA scan enabled

### PR DESCRIPTION
**Pull request by idvoretskyi**
Ref.: https://github.com/cncf/foundation/issues/109

Signed-off-by: Ihor Dvoretskyi <ihor@linux.com>

### CODE REVIEW

**Comment by idvoretskyi** [comment]
cc @stefanprodan @dholbach

File: .github/workflows/fossa.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
```
Comment (stefanprodan) [inline .github/workflows/fossa.yml:4]:
```suggestion
    branches: [main]
```

**Comment by idvoretskyi** [comment]
@stefanprodan fixed, thanks :)

**Comment by stefanprodan** [comment]
@idvoretskyi not quite, pull requests will be made against main not master 😉

**Comment by idvoretskyi** [comment]
@stefanprodan yep, fixed!

**Comment by relu** [comment]
@idvoretskyi would you mind updating your PR once more, maybe squashing the three commits into one and also signing-off so DCO passes? Thank you!

**Comment by squaremo** [comment]
Rebased on main, and into one commit 🐟

**Review by stefanprodan — APPROVED** [review]

File: .github/workflows/fossa.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "^1.14.x"
```
Comment (relu) [inline .github/workflows/fossa.yml:15]: Not sure if important/relevant. Should we _go_ with 1.15? :wink:

Comment (squaremo) [reply .github/workflows/fossa.yml:15]: :-) Let's fix it in post (so we can get Ihor's PR merged and not ask more of them)

**Pull request merged**